cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -129,8 +129,18 @@
         }
         var name = this.id.replace(/^field-/, '');
 
+        var labelField = $this.find('#field-label-' + name);
+
+        if (labelField.parents(".cla-currency-by-frequency").length) {
+          labelField = labelField.parents(".cla-currency-by-frequency").children("legend");
+        }
+
+        var labelText = labelField.text().replace("If yes, ","").replace("If Yes, ","").trim();
```

Comment:
By doing replace twice, are you trying to match upper and lower case "If Yes"

If so you an use a regex to ignore casing when matching:
`replace(/if yes, /i,"") `